readme.md

```diff
@@ -344,6 +344,7 @@
 - [Game Boy Development](https://github.com/avivace/awesome-gbdev#readme)
 - [Construct 2](https://github.com/armaldio/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
+- [Minecraft](https://github.com/bs-community/awesome-minecraft) - Sandbox video game.
```

Comment:
This link should should be https://github.com/bs-community/awesome-minecraft#readme

Reply:
The list seems a bit inconsistent.

- Some of the descriptions end with a period and some do not
- Some of the descriptions are written in Chinese and some are not
	- wiki.vg is missing a description
	- For example, one description says "(Chinese)" written in English

Reply:
Also missing a couple things:

- No project logo
- No code of conduct

Reply:
Is logo necessary?

Reply:
The logo is no necessary but it is strongly suggested.